### DISCUSSION TITLE
Can someone help Loginstruments fix change his license so his modules can be included in Cardinal?

### DISCUSSION THREAD

**Discussion by taji** [DISTRHO/Cardinal#627] [General]
I may have jumped the gun but I wanted to see if Loginstruments excellent LessMess utility module could be included in Cardinal.  I should have asked here first, but wanted to see if he was open to changing his license.  He is, but had some questions that I don't have the knowledge to answer.  Can someone respond here, please?:

https://github.com/LOGUNIVPM/LOGinstruments/issues/18

His LessMess plugin is pretty handy, but he also has a great spectrum analyzer plugin  and a few noise generators as well.

Please let me know if I've gone about this incorrectly or of any other hurdles that need to be over come.

![image](https://github.com/DISTRHO/Cardinal/assets/702031/d3a0cdb4-91df-4a5e-bb52-4ba45c1fa56b)

Thanks!

**Comment by falkTX** [comment]
there is nothing to do license-wise, BSD-3 is very permissive and compatible with gplv3+ as-is already.

author mentions not wanting to see the artwork changed, and we dont usually want to change it (fundamental has copyright/licensing issues).
but we do want to have a dark mode available, either one officially supported by the developer or using cardinal tricks to automatically do it while maintaining the non-monotone colors.

**Reply by dromer** [reply]
I mentioned these points in the ticket.
